atomic_reactor/pulp_util.py

```diff
@@ -10,19 +10,16 @@
 
 import os
 import re
-import logging
 import warnings
 from collections import namedtuple
 
 try:
     import dockpulp
-    from dockpulp import setup_logger
 except (ImportError, SyntaxError):
     dockpulp = None
 
-logger = logging.getLogger(__name__)
-
 PulpRepo = namedtuple('PulpRepo', ['registry_id', 'tags'])
+logger = dockpulp.setup_logger(dockpulp.log)
```

Comment:
We probably need to move this into the `try` block above to handle the Python 3 case (dockpulp is not yet ported to Python 3, so importing it fails there).

Reply:
Fixed, though I had to move it somewhere else to get it to work.